templates/addons/ddb/cf.yml

```diff
@@ -0,0 +1,24 @@
+{{.TableResourceName}}:
+  Type: AWS::DynamoDB::Table
+  Properties:
+    TableName: !Sub ${App}-${Env}-${Name}-{{.TableName}}
+    AttributeDefinitions:{{range .Attributes}}
+      - AttributeName: {{.Name}}
+        AttributeType: {{.DataType}}{{end}}
+    ProvisionedThroughput: 
+      ReadCapacityUnits: 2
+      WriteCapacityUnits: 2
```

Comment:
Maybe we can add `PAY_PER_REQUEST` as the billing mode and get rid of the provisioned throughput.